test/test_view_ops.py

```diff
@@ -1206,9 +1216,18 @@ def test_tensor_split_errors(self, device):
             msg = f'input_size {input_size}, sections_or_indices {sections_or_indices}, dim {dim}'
             with self.assertRaisesRegex(err, err_msg, msg=msg):
                 torch.tensor_split(a, sections_or_indices, dim)
+            with self.assertRaisesRegex(err, err_msg, msg=msg):
+                torch.tensor_split(a, torch.tensor(sections_or_indices), dim)
             with self.assertRaises(numpy_err, msg=msg):
                 np.array_split(a.cpu().numpy(), sections_or_indices, dim)
 
+        # test tensor indices being a 1-dimension tensor of dtype long
```

Comment:
This section of the tests will need to be updated to reflect the error checking changes mentioned above